### PR TITLE
Create FMI_Project_news.md

### PR DESCRIPTION
**Pull request by chrbertsch**

### CODE REVIEW

File: _2019-03/FMI_Project_news.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+title: News from the FMI Project
+author: MAP FMI
+category: "project"
```
Comment (beutlich) [inline _2019-03/FMI_Project_news.md]:
```suggestion
category: "association"
```

File: _2019-03/FMI_Project_news.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+---
+
+## FMI 2.0.1 released
+
+After approval by the FMI Steering Committee and the Modelica Association members, version 2.0.1 of the FMI Specification 
+has been released today.
```
Comment (beutlich) [inline _2019-03/FMI_Project_news.md]: No relative dates. Please use absolute date here.

File: _2019-03/FMI_Project_news.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+## Towards FMI 3.0
+
+After acceptance by the FMI Steering committee, an “extra” directory for meta information has been added to the 
+[FMI 3.0 features list in FMI FAQ](https://fmi-standard.org/faq/).
+All FCPs (FMI Change proposals) described in the features list have now already been merged into the master branch of the 
+[FMI standard repostory](https://github.com/modelica/fmi-standard). With this step, the API of the coming FMI 3.0 standard has been 
+stabilized, so that prototyping of tools and improvement of the standard text can start now.
+
+The development of the FMI standard can be followed [on Github](https://github.com/modelica/fmi-standard) for the sources
```
Comment (beutlich) [inline _2019-03/FMI_Project_news.md]:
```suggestion
The development of the FMI standard can be followed on [GitHub](https://github.com/modelica/fmi-standard) for the sources 
```

File: _2019-03/FMI_Project_news.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+[FMI 3.0 features list in FMI FAQ](https://fmi-standard.org/faq/).
+All FCPs (FMI Change proposals) described in the features list have now already been merged into the master branch of the 
+[FMI standard repostory](https://github.com/modelica/fmi-standard). With this step, the API of the coming FMI 3.0 standard has been 
+stabilized, so that prototyping of tools and improvement of the standard text can start now.
+
+The development of the FMI standard can be followed [on Github](https://github.com/modelica/fmi-standard) for the sources 
+of the standard text, headers and schema files, see also  the [generated documentation](https://fmi-standard.org/docs/3.0-dev/) for
+the latest devlopment version.
+You are welcome to sign the [CLA](https://svn.fmi-standard.org/fmi/branches/public/FMI_CCLA_v1.0_2016_06_21.pdf) of the FMI project 
+or the [CCLA](https://www.modelica.org/licenses/ModelicaAssociationCLA_1.1) fo the Modelica Assciation and comment and contribute
```
Comment (beutlich) [inline _2019-03/FMI_Project_news.md]:
```suggestion
or the [CCLA](https://www.modelica.org/licenses/ModelicaAssociationCLA_1.1) of the Modelica Assciation and comment and contribute 
```

File: _2019-03/FMI_Project_news.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+## Towards FMI 3.0
+
+After acceptance by the FMI Steering committee, an “extra” directory for meta information has been added to the 
+[FMI 3.0 features list in FMI FAQ](https://fmi-standard.org/faq/).
+All FCPs (FMI Change proposals) described in the features list have now already been merged into the master branch of the 
+[FMI standard repostory](https://github.com/modelica/fmi-standard). With this step, the API of the coming FMI 3.0 standard has been 
+stabilized, so that prototyping of tools and improvement of the standard text can start now.
+
+The development of the FMI standard can be followed on [Github](https://github.com/modelica/fmi-standard) for the sources 
+of the standard text, headers and schema files, see also  the [generated documentation](https://fmi-standard.org/docs/3.0-dev/) for
```
Comment (beutlich) [inline _2019-03/FMI_Project_news.md]:
```suggestion
of the standard text, headers and schema files, see also the [generated documentation](https://fmi-standard.org/docs/3.0-dev/) for
```

File: _2019-03/FMI_Project_news.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+the latest devlopment version.
+You are welcome to sign the [CLA](https://svn.fmi-standard.org/fmi/branches/public/FMI_CCLA_v1.0_2016_06_21.pdf) of the FMI project 
+or the [CCLA](https://www.modelica.org/licenses/ModelicaAssociationCLA_1.1) fo the Modelica Assciation and comment and contribute
```
Comment (beutlich) [inline _2019-03/FMI_Project_news.md]:
```suggestion
or the [CLA](https://www.modelica.org/licenses/ModelicaAssociationCLA_1.1) of the Modelica Assciation and comment and contribute 
```

File: _2019-03/FMI_Project_news.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+
+## Towards FMI 3.0
+
+After acceptance by the FMI Steering committee, an “extra” directory for meta information has been added to the 
+[FMI 3.0 features list in FMI FAQ](https://fmi-standard.org/faq/).
+All FCPs (FMI Change proposals) described in the features list have now already been merged into the master branch of the 
+[FMI standard repostory](https://github.com/modelica/fmi-standard). With this step, the API of the coming FMI 3.0 standard has been 
+stabilized, so that prototyping of tools and improvement of the standard text can start now.
+
+The development of the FMI standard can be followed on [Github](https://github.com/modelica/fmi-standard) for the sources
```
Comment (beutlich) [inline _2019-03/FMI_Project_news.md]:
```suggestion
The development of the FMI standard can be followed on [GitHub](https://github.com/modelica/fmi-standard) for the sources 
```

**Comment by TorstenBlochwitz** [comment]
I would propose to Exchange the first 2 sentences, like this:

All FMI Change Proposals described in the [FMI 3.0 features list] (https://fmi-standard.org/faq/) have now already been merged into the master branch of the [FMI standard repository](https://github.com/modelica/fmi-standard). With this step, the API of the coming FMI 3.0 standard has been stabilized, so that prototyping of tools and improvement of the standard text can start now.
At their meeting in October in Lund the FMI Steering Committee accepted one new feature for FMI 3.0: the FMU ZIP Archive can now contain an additional folder with meta data to travel with the FMU, which can be modified by different tools, allowing for layered standards.

And I would propose to exchange the last two sentences too:

You are welcome to comment and contribute to the FMI 3.0 standard. Please note: you have to sign the [CCLA](https://svn.fmi-standard.org/fmi/branches/public/FMI_CCLA_v1.0_2016_06_21.pdf) of the FMI project or the [CLA](https://www.modelica.org/licenses/ModelicaAssociationCLA_1.1) of the Modelica Association in order to do so.

**Pull request merged**